devops/business_services/new_customer.rst

```diff
@@ -0,0 +1,72 @@
+Create new Customer
+===================
+
+Add Customer module
+-------------------
+
+:ref:`add_customer_module`
+
+Create Database/Storage
+-----------------------
+
+Postgres Database
+^^^^^^^^^^^^^^^^^
+
+Add the new databases to the `VSHN git Repository`_.
+The database will automatically be created after a few minutes.
+
+.. _VSHN Git Repository: https://git.vshn.net/tocco/tocco_hieradata/edit/master/database/master.yaml
+
+| Use the same password for both databases of the installation
+| This password is also used to setup OpenShift.
```

Comment:
> This password is also used to setup OpenShift.

I'd be a bit more specific, like "This password needs to be used as `DB_PASS` parameter to `oc process` later on."